templates/site/js/dataproduct-form.js

```diff
@@ -1367,4 +1367,32 @@ async function submitForm(){
     submitObject["authors"] = authorArray;
     submitObject["metadata"] = metadataObject;
     submitObject["path"] = linkedPathField.value;
+
+    var marbleAPIURL = "{{ configs['node_details']['marble_api_path'] }}";
+    var submitErrorElement = document.getElementById("submitError");
+
+    try {
+        const response = await fetch(marbleAPIURL, {
+            method: "POST",
+            headers: {
+                "Accept": "application/json",
+                "Content-Type":"application/json"
+            },
+            body:JSON.stringify(submitObject)
+        });
+
+        const result = await response.json();
+
+        if (!response.ok) {
```

Comment:
typically if you have an if/else statement you don't negate the condition, just switch the blocks. It's easier to read.